app/controllers/people_controller.rb

```diff
@@ -184,10 +184,15 @@ def aspect_membership_dropdown
   private
 
   def find_person
-    @person = Person.find_from_guid_or_username({
-      id: params[:id] || params[:person_id],
-      username: params[:username]
-    })
+    username = params[:username]
+    if diaspora_id?(username)
+      @person = Person.where(:diaspora_handle => username.downcase).first
+    else
+      @person = Person.find_from_guid_or_username({
+        id: params[:id] || params[:person_id],
+        username: username
+      })
+    end
```

Comment:
While we have no Ruby styleguide yet, please at least decide on a hash form to use in a single diff hunk ;)